doc/api/debugger.md

```diff
@@ -187,12 +187,15 @@ flag instead of `--inspect`.
 $ node --inspect index.js
 Debugger listening on 127.0.0.1:9229.
 To start debugging, open the following URL in Chrome:
-    chrome-devtools://devtools/bundled/inspector.html?experiments=true&v8only=true&ws=127.0.0.1:9229/dc9010dd-f8b8-4ac5-a510-c1a114ec7d29
+    chrome-devtools://devtools/bundled/js_app.html?experiments=true&v8only=true&ws=127.0.0.1:9229/dc9010dd-f8b8-4ac5-a510-c1a114ec7d29
 ```
 
 (In the example above, the UUID dc9010dd-f8b8-4ac5-a510-c1a114ec7d29
 at the end of the URL is generated on the fly, it varies in different
 debugging sessions.)
 
+> If the Chrome browser is older than <strong>66.0.3345.0</strong>,
+>use `inspector.html` instead of `js_app.html` in the above URL.
```

Comment:
Would `chrome://inspect` work in all versions? If so, maybe it's easier to recommend using that URL than doing string substitution on the URL.

Reply:
In Chromium <strong>51.0.2704.0</strong>, `chrome://inspect` doesn't provide the link to open inspector.html.


![screen shot 2018-06-23 at 6 11 38 pm](https://user-images.githubusercontent.com/10280768/41809780-f561de12-7710-11e8-8a42-b277b9fabd2f.png)


Whereas, a more recent version <strong>67.0.3396.87</strong> has it.


![screen shot 2018-06-23 at 6 07 04 pm](https://user-images.githubusercontent.com/10280768/41809770-c26549a4-7710-11e8-84c0-06b4f30224df.png)